packages/verification-vue-component/src/style/components/verification/card/_faq.scss

```diff
@@ -0,0 +1,60 @@
+.faq-card {
+    .card-header {
+        padding: 7px 4px;
+        font-weight: bold;
+    }
+
+    .card-body {
+        .faq-items {
+            overflow: hidden;
+
+            .item {
+                margin: 4px;
+                overflow: hidden;
+                border-radius: 6px 4px;
+                box-shadow: 0 2px 12px rgba(32, 51, 70, 0.09), 0 1px 4px rgba(32, 51, 70, 0.04);
```

Comment:
In SASS you can also use HEX values or variables in rgba, like this: rgba(hex, opacity)

Reply:
Didn't now that, changed it